protostar/commands/test/test_scheduler.py

```diff
@@ -28,9 +29,16 @@ def run(
         include_paths: List[str],
         is_account_contract: bool,
         disable_hint_validation: bool,
+        exit_first: bool,
     ):
         with multiprocessing.Manager() as manager:
-            test_results_queue = TestResultsQueue(manager.Queue())
+            test_results_queue = TestResultsQueue(
+                manager.Queue(),
+                manager.Value(
+                    ctypes.c_bool,
+                    (len(test_collector_result.broken_test_suites) > 0) and exit_first,
+                ),
+            )
```

Comment:
It's hard to tell what's the purpose of this value. Please pass it as a keyword argument.